feat(headless): migrate headless rga document from docs site to typedoc (#6838)
coveo/ui-kit: samples/headless/headless-rga-angular/README.md

# Headless RGA with Angular

This sample demonstrates a minimal set of interactions with Headless RGA in the Angular framework.

## Technology Stack

- **Angular**: UI framework
- **@coveo/headless**: Coveo's Headless library
- **TypeScript**: Type-safe JavaScript

## Prerequisites

- Node.js 22+ (LTS recommended)
- pnpm 10+

## Getting Started

1. Install dependencies:
   ```bash
   pnpm install
   ```
2. Start the development server:
   ```bash
   pnpm run start
   ```

3. Open your browser to `http://localhost:5173`

## Available Scripts

```bash
pnpm run dev       # Start development server
pnpm run build     # Build for production
```

## Usage

> When using the configuration from `getSampleSearchEngineConfiguration`, you will only be able to return a response for the query `what is ipx?`.

Reviewer: ```suggestion
When using the configuration from `getSampleSearchEngineConfiguration`, you can, for example, receive a response for the query `what is ipx?`.
```
Because I think that others might work too, although they're not easy to find.

Author: I think its best to be declarative here and say "this will work" as really it doesn't matter if other things will work - such as that "tivo" query in use elsewhere - we want to direct users to a golden path in these samples.
